data/articles/09560.json

```diff
@@ -8021,49 +8021,94 @@
                     "type": "paragraph"
                 }
             ],
-            "decisionLetter": {
+            "elifeAssessment": {
```

Comment:
Let's leave 09560 alone and create a new article with all the new sections and which we are at it make the new article have an `eLocationId` with a prefix of `RP` so that we can begin to treat this article as a full VoR PRC example.